config/chime_science_run_recv.yaml

```diff
@@ -267,6 +266,28 @@ remove_ev:
     out_buf: visbuf_10s_timing_ne
 
 
+# This stage is here to track the output dataset IDs in prometheus
+vis_dset_tracking:
+  n2_mf:
+    kotekan_stage: visDebug
+    in_buf: visbuf_10s_mfreq
+  chimestack:
+    kotekan_stage: visDebug
+    in_buf: visbuf_10s_stack_ne
+  26m:
+    kotekan_stage: visDebug
+    in_buf: visbuf_5s_26m_ungated_post
+  26mgated:
+    kotekan_stage: visDebug
+    in_buf: visbuf_5s_26m_gated_post
+  cal:
+    kotekan_stage: visDebug
+    in_buf: visbuf_10s_cal
+  timing:
+    kotekan_stage: visDebug
+    in_buf: visbuf_10s_timing_ne{
```

Comment:
```suggestion
    in_buf: visbuf_10s_timing_ne
```

Reply:
I think you and @nritsche both caught the same thing. It's vim's fault.